seshdash/views.py

```diff
@@ -96,21 +96,27 @@ def index(request,site_id=0):
     # Create an object of the get_high_chart_date
     context_dict['high_chart']= get_high_chart_data(request.user,site_id,sites)
     context_dict['site_id'] = site_id
-    #Generate measurements in the time_series_graph
-    client=Influx()
-    measurements_value=client.get_measurements()
-    measurements =[]
-
-    if measurements_value is not None:
-        for measurement in measurements_value:
-            measurements.append(measurement['name'])
 
+    #Generating site measurements for a graph
+    current_site = Sesh_Site.objects.filter(id = site_id).first()
+    # getting site measurements
+    site_measurements = get_site_measurements(current_site)
+
+    measurements ={}
+    # verbose names names for measurements
+    measurements_verbose_name = dict(Site_Measurements.ROW_CHOICES)
+    for measurement in site_measurements:
```

Comment:
Man this is a really bad way of doing this. This will throw so many errors!
Why not just iterate over each one and check if it matches?

This can be done with a quick lambda.